### ISSUE TITLE
docs: Does CLI `add` command support --dryRun and --force ?

### ISSUE THREAD

**Issue by jinbeomhong** [angular/angular#28539]
# 📚 Docs or angular.io bug report

### Description

I think `ng add` command doesn't support `--dryRun` flag, because it isn't working. A command below actually installs a package into my project:

```bash
$ ng add @nebular/theme --dryRun
Installing packages for tooling via npm.
npm WARN @nebular/theme@3.2.1 requires a peer of @angular/cdk@^7.2.1 but none is installed. You must install peer dependencies yourself.
npm WARN @nebular/theme@3.2.1 requires a peer of nebular-icons@^1.1.0 but none is installed. You must install peer dependencies yourself.

+ @nebular/theme@3.2.1
added 2 packages from 2 contributors and audited 41442 packages in 8.136s
found 0 vulnerabilities
...
```

You can also see that there isn't `--force` flag:

```bash
$ ng add --help
arguments:
  collection
    The package to be added.

options:
  --defaults
    When true, disables interactive input prompts for options with a default.
  --help
    Shows a help message for this command in the console.
  --interactive
    When false, disables interactive input prompts.
```

But the `--dryRun`, `--force` flag appears at docs.

### What's the affected URL?**

https://angular.io/cli/add

## 📷Screenshot

<img width="754" alt="screenshot2" src="https://user-images.githubusercontent.com/32840866/52266772-8d9aeb80-297a-11e9-89ff-3866024b09cb.png">

## 🔥 Exception or Error
<pre><code>

</code></pre>

**Comment by gkalpak** [comment]
This has already been fixed on master (see https://next.angular.io/cli/add) and will be fixed on stable with #28467.

**Comment by gkalpak** [comment]
Thx for reporting this.
The issue has already been fixed on master (see https://next.angular.io/cli/add) and will be fixed on stable with #28467.
